R/html_dependency.R

```diff
@@ -231,32 +231,37 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
     }
   }
 
-  if (!file.exists(outputDir))
+  if (!dir_exists(outputDir))
     dir.create(outputDir)
```

Comment:
I'm not sure if this suppresses other possible warnings as well. If it does, I'd rather stick to the current approach.